src/Formatter/ClassNameFormatter.php

```diff
@@ -26,6 +26,11 @@ public function commandHandled($command)
         return 'Command succeeded: ' . get_class($command);
     }
 
+    public function propertiesUsed($command)
```

Comment:
I'm not super sold on this name and if we add the exception info to the context, it'll probably change as well. getLoggerContext() or extra info or some such?